telos-frontend/src/components/top-bar/TopBar.jsx

```diff
@@ -0,0 +1,66 @@
+import AppBar from '@material-ui/core/AppBar';
+import { makeStyles } from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import SettingsIcon from '@material-ui/icons/Settings';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+
+// Used to define styles of MUI components.
+const useStyles = makeStyles(() => ({
```

Comment:
Nice clean styling, however, could this be pulled out into a CSS file to separate concerns?

Reply:
My understanding is that this is the way the hook api for styling mui components works. I don't think it can be abstracted out to a css file without changing the way we are styling these components.